MANIFEST.in

```diff
@@ -3,9 +3,12 @@ include CONTRIBUTING.rst
 include HISTORY.rst
 include LICENSE
 include README.rst
+include Makefile
+
+exclude .editorconfig requirements_dev.txt
 
 recursive-include tests *
-recursive-exclude * __pycache__
-recursive-exclude * *.py[co]
+recursive-include docs *.rst conf.py make.bat Makefile
 
-recursive-include docs *.rst conf.py Makefile make.bat *.jpg *.png *.gif
+recursive-exclude * __pycache__
```

Comment:
See global exclude syntax in https://github.com/azavea/django-amazon-ses/commit/f4903b2a711287884b3244a5e4f461c723c2ad5f.